src/SFA.DAS.FAT.Web/Views/CourseProviders/CourseProviderDetails.cshtml

```diff
@@ -632,6 +632,9 @@
                 </div>
             </div>
         </div>
+        <div class="govuk-grid-column-full">
```

Comment:
I think we should not have the same link hard coded in multiple places. Can you create a constant and bind the link via the viewmodel.

Reply:
If the link is same in terms of content and outlook then there is option to create a partial as well.